### PR TITLE
Modified readme according to #312.

### LINKED ISSUE #312: [📚 Documentation Issue]: Improving the readme
### Preliminary Checks

- [X] I have read the relevant section of the documentation.
- [X] I have searched the issue tracker for similar documentation issues.

### Documentation Section

Readme

### Issue Description

Readme can be made more clear with the following changes:
- Start with something catchy: "Can someone explain to me why eurocode is not publicly available in code and that all of us are coding eurocode with massive maintenance cost and shortages in technical personel? Well, worry no longer!"
- In the first line of the introduction replace "." with "!"
- Remove the tekst of the introduction after the first sentence. 
- After the first sentence start explicit description of what blueprints is. e.g. "Blueprints is a collaboration of several engineering companies which contains programmable eurocode information such as tables, figures and formulas. It also provides materials, geometry and even section checks. The basis as well as the implemented usecases provide all key necessities for the civil engineer." 
- remove the entire Vision section
- make a separate header called 'Read the docs!' for the link to the doc which now is placed under installation. 

### Confirmation

- [X] I have checked that this issue does not already exist in the issue tracker.
- [X] I have read and understood the contribution guidelines.

### PR DESCRIPTION
**Pull request by Simone-de-Rijke**
## Description

Edited readme according to 312.
314 will follow later. 

Fixes #312 

## Checklist:

- [X] I have added tests that prove my fix is effective or that my feature works
- [X] I have commented my code, particularly in hard-to-understand areas
- [X] I have made corresponding changes to the documentation
- [X] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -13,23 +13,22 @@
   <img alt="blueprints banner" src="docs/source/_static/blueprints_banner.png">
 </picture>
 
-Welcome to Blueprints, the cornerstone repository for civil engineering professionals and enthusiasts alike. Blueprints is a collaborative,
-open-source initiative designed to serve as a comprehensive library of code, tools, and best practices, aiming to streamline and innovate the
-workflows within the built environment sector.
+Can someone explain to me why eurocode is not publicly available in code and that all of us are coding eurocode with massive maintenance costs 
+and shortages in technical personel? Well, no longer!
```
Comment (RamiEvans) [inline README.md:17]: Spelling mistake - personnel

File: README.md [hunk 2]
```diff
@@ -13,23 +13,22 @@
   <img alt="blueprints banner" src="docs/source/_static/blueprints_banner.png">
 </picture>
 
-Welcome to Blueprints, the cornerstone repository for civil engineering professionals and enthusiasts alike. Blueprints is a collaborative,
-open-source initiative designed to serve as a comprehensive library of code, tools, and best practices, aiming to streamline and innovate the
-workflows within the built environment sector.
+Can someone explain to me why eurocode is not publicly available in code and that all of us are coding eurocode with massive maintenance costs
```
Comment (RamiEvans) [inline README.md:16]: eurocode - Should probably be Eurocode with capital

File: README.md [hunk 3]
```diff
@@ -13,23 +13,22 @@
...
+and shortages in technical personel? Well, no longer!
 
-## Vision
+Welcome to Blueprints, the cornerstone repository for civil engineering professionals and enthusiasts alike!
 
-Our vision with Blueprints is to create an accessible, reliable, and community-driven resource that supports engineers, and developers in their daily
-challenges. By harnessing the collective expertise of the industry, Blueprints strives to pave the way for a future where collaboration and open
-source can drive the field of civil engineering to new heights.
+Blueprints is a collaboration of several engineering companies which contains programmable eurocode information such as tables, figures and 
+formulas. It also provides materials, geometry and even section checks. The basis as well as the implemented usecases provide all key
```
Comment (RamiEvans) [inline README.md:22]: usecases - Spelling mistake - use cases

**Review by RamiEvans — CHANGES_REQUESTED** [review]
Looks good overall. Slight spelling / grammar issues

File: README.md (hunk 1 above)
Comment (Simone-de-Rijke) [reply README.md]: Corrected in 0b05c4961945a3e056a325b506274a867b82ff6a

File: README.md (hunk 2 above)
Comment (Simone-de-Rijke) [reply README.md]: Corrected in 0b05c4961945a3e056a325b506274a867b82ff6a

File: README.md (hunk 3 above)
Comment (Simone-de-Rijke) [reply README.md]: Corrected in 0b05c4961945a3e056a325b506274a867b82ff6a

File: README.md (hunk 2 above)
Comment (RamiEvans) [reply README.md]: Second Eurocode in sentence

Comment (Simone-de-Rijke) [reply README.md]: And there was a third one! :) corrected it here b3738fa90475bd6bbe4049165e16cb5dcaa567ef

**Review by RamiEvans — APPROVED** [review]
All comments closed and adjustments made to general grammar and spelling

**Pull request merged**